Sources/ReferenceResolver.swift

```diff
@@ -5,7 +5,7 @@ protocol ResolvableType: Builder {
 
 private let kSchemaBuilderReferenceResolver = ReferenceResolver<SchemaBuilder>()
 extension SchemaBuilder: ResolvableType {
-    static var path: String { return "definitions" }
+    static var path: String { return "schemas" }
```

Comment:
The only different path in OAS2 vs OAS3 (except new/dropped) ones - we could add some versioning by changing it to func accepting swagger/version or possibly easier would be to create some duplicate of it like 'DefinitionBuilder'?